### ISSUE TITLE
Training Problem

### ISSUE THREAD

**Issue by alanWXZ** [yanx27/2DPASS#11]
Thank you for your excellent work! When I run the code, the performance of the first four epochs are 51.682 mIoU, 49.632 mIoU, 50.934 mIoU and 47.684 mIoU. The performance is decreasing with training. I don not know  that if it is a reasonable experimental result.

**Comment by yanx27** [comment]
@alanWXZ  Hi, could you give more details such as your environment or training setup? During my experiment (default setup with batch size 8 on a V100), it will achieve finally over 66 mIoU (about 69 after TTA) in 64 epochs, as shown in the figure.  It will keep a long time in the range of 50-60 mIoU since the nature of cosine learning rate scheduler. I also show the Tensorboard snapshot here if you want to check the performance.

![image](https://user-images.githubusercontent.com/35829331/192137202-afc6bf9f-d03f-4972-bf52-14d5a61bf4f9.png)
